ui/ui-frontend/projects/vitamui-library/src/sass/_button.scss

```diff
@@ -45,8 +45,8 @@
       color: $button-color;
     }
 
-    &:hover,
-    &:focus {
+    &:active,
+    &:hover {
```

Comment:
Le soucis, c'est qu'on ne voit plus qu'on est en focus sur un bouton en navigation au clavier.
Et tant qu'à faire, on pourrait peut-être s'aligner avec [le Figma](https://www.figma.com/design/YHKmbbnI25ttjMLBUhXu3u/Vitam-UI-%F0%9F%A7%A9-Design-System?node-id=628-35&m=dev) pour implémenter correctement le hover (couleur plus claire) et le focus (mais du coup on serait en mode "focus" après un clic...) :
![image](https://github.com/user-attachments/assets/4e950242-da8b-4ec6-bfce-e0b414adbd38)

Reply:
Une possibilité est d'appliquer le style `focus` que si l'élément n'est pas `active` (donc le style `focus` ne s'applique pas pendant le clic) et de retirer le focus du composant bouton au moment du clic (donc le style `focus` ne s'applique pas non plus après le clic). Dans ce cas, le style `focus` ne s'applique qu'à la navigation au clavier.
Donc :
- un style sur `:focus:not(:active)`
- un `$event.target.blur();` au moment du clic sur le button

Reply:
le figma n'a pas été revu sur cette partie.      

et je pense que ce qu'il faudrait c'est le pas mettre de outline none.      
et laisser au navigateur le soin de la navigation clavier.